packages/amplify-provider-awscloudformation/src/override-manager/amplify-factory-transform.ts

```diff
@@ -0,0 +1,31 @@
+import { IAmplifyResource, AmplifyCategoryTransform, $TSContext, Template } from 'amplify-cli-core';
+
+const categoryCfnTransformMap = (resourceName: string) => ({
+  /**
+  * auth: {
+    Cognito: new AmplifyAuthTransform(resourceName)
+  }
+   *  add Factories here for overrides
+   */
+  default: {
+    service: new AmplifyNoneTransform(resourceName),
+  },
+});
+
+export class AmplifyCategoryTransformFactory {
+  public static getCategoryTransformInstance(resource: IAmplifyResource): AmplifyCategoryTransform {
+    return categoryCfnTransformMap(resource.resourceName)[resource.category][resource.service];
+  }
+}
+
+export class AmplifyNoneTransform extends AmplifyCategoryTransform {
```

Comment:
Why is it named AmplifyNoneTransform? What's the purpose of this class?

Reply:
removed